manifests/init.pp

```diff
@@ -284,84 +284,119 @@
       $package_name_default      = 'nscd'
       $package_source_default    = undef
       $service_name_default      = 'nscd'
-      case $::operatingsystemmajrelease {
-        '5': {
```

Comment:
Now we know  that at least two versions of Amazon linux are compatible to EL 7 (actually nscd version almost the same).
So it will work till next version. :)
In next version we can easily check if something new were added and  fix it if needed.

The only one problem that we weren't able to support older version of Amazon linux, but I'm not sure that we need.

By default Amazon installs updates in new machine and they suggest to use the latest AMI.
I don't want to believe that someone is still using AmL 2011.

On Tue, 2016-04-05 at 09:11 -0700, Garrett Honeycutt wrote:
In manifests/init.pp<https://github.com/ghoneycutt/puppet-module-nscd/pull/42#discussion_r58567269>:



> @@ -284,84 +284,119 @@
>        $package_name_default      = 'nscd'
>        $package_source_default    = undef
>        $service_name_default      = 'nscd'
> -      case $::operatingsystemmajrelease {
> -        '5': {


Not sure how we would support this if there is not a mapping between versions.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/ghoneycutt/puppet-module-nscd/pull/42/files/b3fc20c1e5a5f46868951ac9eb8fb6fd60f4a963#r58567269>